packages/webdriver-image-comparison/src/helpers/options.ts

```diff
@@ -32,7 +33,7 @@ export function defaultOptions(options: ClassOptions): DefaultOptions {
         fullPageScrollTimeout: options.fullPageScrollTimeout ?? FULL_PAGE_SCROLL_TIMEOUT,
         hideScrollBars: Object.prototype.hasOwnProperty.call(options, 'hideScrollBars')
             ? Boolean(options.hideScrollBars)
-            : true,
+            : isStorybook() ? false : true,
```

Comment:
same, can we add a comment why we story book is special here?

Reply:
Good feedback, will do that